### DISCUSSION TITLE
Setting connection_name not working

### DISCUSSION THREAD

**Discussion by hopewise** [rabbitmq/rabbitmq-server#11109] [Other] [answered]
### Describe the bug

I am trying to set the connection_name for better visibility of connections on the RabbitMQ. Currently the connections tab shows IPs only and undefined for the provided `connection_name`

### Reproduction steps

```
connection_settings = {
  host: Settings.rabbitmq.host,
  port: Settings.rabbitmq.port,
  user: Settings.rabbitmq.user,
  password: Settings.rabbitmq.password,
  ssl: true,
  heartbeat: 60, # will use RabbitMQ setting
  conn_per_thread: true,
  conn_retry_attempts: 5,
  verify_peer: true,
  frame_max: 131072,
  auth_mechanism: "PLAIN",
  connection_name: "AGG"
}
Bunny_Connection = Bunny.new connection_settings
```

or use 
```
  client_properties: {connection_name: "AGG"}
```

### Expected behavior

To see connection name `AGG` under the IP address and in the connection details:

<img width="349" alt="Screenshot 2024-04-28 at 1 31 50 PM" src="https://github.com/rabbitmq/rabbitmq-dotnet-client/assets/1061157/6ac9bed8-6895-4726-b03d-6126fad84d0c">

**Comment by michaelklishin** [comment]
@hopewise the code provided suggests you use [Bunny](https://github.com/ruby-amqp/bunny). Why was it filed as an issue for the RabbitMQ .NET client?

Moved to `rabbitmq/rabbitmq-server` discussions.

**Comment by michaelklishin** [comment, marked as answer]
``` ruby
conn = Bunny.new(connection_name: "app ABC #{rand}")
conn.start
```

Works just fine for me on the latest versions of both Bunny and RabbitMQ. `connection_name` support was introduced to Bunny [in `2.17.0` in 2020](https://github.com/ruby-amqp/bunny/pull/600) and hasn't changed since.

Here's evidence for the above connection.

![Screenshot by Dropbox Capture](https://github.com/rabbitmq/rabbitmq-server/assets/1090/5addfcd6-8905-4a60-8f8b-477a41c831bf)
